### PR TITLE
feat: Implement stylelint-scss & version bumps

### PR DESCRIPTION
**Pull request by reintroducing**
**Issue Link**
https://spothero.atlassian.net/browse/FED-241

**Description**
Title.

### CODE REVIEW

File: rules/scss.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+module.exports = {
```
Comment (hnipps) [inline rules/scss.js:1]: Does this change any of our existing rules?

Comment (reintroducing) [reply rules/scss.js:1]: these are all rules that used to exist in our codebase but got removed 2 years ago accidentally when i moved over to using single files for each set of rules with this commit: https://github.com/spothero/stylelint-config/commit/c21fbe616e3fa2ff851d913f2f4ffba3574370d3

You will see i removed the scss.js file and never accounted for it in the combined files even though we still had stylelint-scss plugin in the package.json. this wasn't intentional so i wanted to re-add them back in. i dont doubt there will be some updates necessary to conform throughout the projects but hopefully theyll be minimal.

Comment (hnipps) [reply rules/scss.js:1]: Awesome, thank you.

File: package.json [hunk 2]
```diff
@@ -6,6 +6,15 @@
   "repository": "https://github.com/spothero/stylelint-config",
   "author": "SpotHero",
   "license": "Apache-2.0",
+  "eslintConfig": {
```
Comment (sethomas) [inline package.json:9]: +1

**Pull request merged**